resources/assets/js/components/people/conversation/Conversation.vue

```diff
@@ -0,0 +1,127 @@
+<style scoped>
```

Comment:
The style should be moved to sass files

Reply:
No. This is the principle of having components - styles that applied to a component should be kept in this component.

Reply:
Hum, unless we need to apply `@if ltr` everywhere...